keps/prod-readiness/sig-storage/1432.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 1432
 alpha:
   approver: "@deads2k"
+beta:
+  approver: "@wojtek-t"
```

Comment:
I'm sorry - I guess I introduced some confusion.

Please leave it assigned to @deads2k - we're trying to not change approvers in the middle if not really needed.
My point was just that you need a new entry for every stage, not replace the already existing ones.